liblangutil/Token.cpp

```diff
@@ -72,6 +72,9 @@ void ElementaryTypeNameToken::assertDetails(Token _baseType, unsigned const& _fi
 			"No elementary type " + string(TokenTraits::toString(_baseType)) + to_string(_first) + "x" + to_string(_second) + "."
 		);
 	}
+	else
+		solAssert(_first == 0 && _second == 0, "Address should not have size arguments");
```

Comment:
The comment seems to be wrong?

Reply:
updated.
@chriseth why is it okay to expect 0 for both here?